Library/Formula/shibboleth-sp.rb

```diff
@@ -0,0 +1,109 @@
+class ShibbolethSp < Formula
+  desc "Shibboleth 2 Service Provider daemon"
+  homepage "https://wiki.shibboleth.net/confluence/display/SHIB2"
+  url "http://shibboleth.net/downloads/service-provider/latest/shibboleth-sp-2.5.5.tar.gz"
+  sha256 "30da36e0bba2ce4606a9effc37c05cd110dafdd6d3141468c4aa0f57ce4d96ce"
+
+  option "with-homebrew-httpd22", "Use Homebrew Apache httpd 2.2"
+  option "with-homebrew-httpd24", "Use Homebrew Apache httpd 2.4"
+
+  depends_on "curl" => "with-openssl"
+  depends_on "opensaml"
+  depends_on "xml-tooling-c" => "with-openssl"
+  depends_on "xerces-c"
+  depends_on "xml-security-c"
+  depends_on "log4shib"
+  depends_on "boost"
+  depends_on "homebrew/apache/httpd22" if build.with? "homebrew-httpd22"
+  depends_on "homebrew/apache/httpd24" if build.with? "homebrew-httpd24"
+
+  skip_clean "var/run/shibboleth"
```

Comment:
I don't think this is something I can get around. When shibboleth first starts it needs var/cache/shibboleth and var/run/shibboleth to exist so it can copy some files to cache and create a pid file in run. I believe (I may be wrong about this though) that shibboleth looks for these directories with a path relative to the shibd executable. I know that using skip clean is a working solution at the current moment, and if I can figure out a way to get rid of it I certainly will.